process/scToProtocol/stakingToPeer.go

```diff
@@ -132,7 +132,7 @@ func (stp *stakingToPeer) UpdateProtocol(body block.Body, nonce uint64) error {
 		return err
 	}
 
-	for key := range affectedStates {
+	for _, key := range affectedStates {
```

Comment:
what a bug :1st_place_medal:

Reply:
yes, it was. needed list instead of map.